misc/scripts/search.sh

```diff
@@ -143,7 +143,7 @@ while IFS= read -r URL; do
             || fancy_message warn "Replace '~' with the full home path on \e]8;;file://$STGDIR/repo/pacstallrepo\a$CYAN$STGDIR/repo/pacstallrepo$NC\e]8;;\a"
         URL="${URL/'~'/$HOME}"
     fi
-    if ! check_url "${URL}/packagelist"; then
+    if [[ ${URL} != "#"* ]] && ! check_url "${URL}/packagelist"; then
```

Comment:
This would fail if for some reason the string was `   # URL`.

Reply:
That's the entire purpose of this. Skipping commented out REPOS from the file.

Reply:
I mean there should allow an arbitrary amount of whitespace before the `#`.